jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBQueryResultSet.java

```diff
@@ -1172,7 +1210,7 @@ public boolean wasNull() throws SQLException {
   }
 
   private void checkRecord() throws SQLException {
-    if (record == null) {
+    if (Objects.isNull(record)) {
```

Comment:
the record could not be null, You already init this field in the constructor of this class. Remove this check